.rubocop.yml

```diff
@@ -90,6 +99,11 @@ Metrics/CyclomaticComplexity:
   Enabled: true
   Max: 12
 
+Metrics/ParameterLists:
+  Enabled: true
+  Exclude:
+    - "spec/**/*.rb"
+
```

Comment:
we use a unified config - this file is synced from dry-rb/template-gem so it would be good to update it there so that all repos are in sync